### PR TITLE
Task #223475: cohort list no data condition handling,add loading for logout

### PR DESCRIPTION
**Pull request by upendraTekdi**
Task #223475: cohort list no data condition handling,add loading for logout

### CODE REVIEW

File: src/pages/cohorts.tsx [hunk 1]
```diff
@@ -28,55 +33,62 @@ type UserDetails = {
...
     React.useState<boolean>(false);
   const [inputName, setInputName] = React.useState<string>("");
+  const [cohortName, setCohortName] = React.useState<string>("");
+  const [loading, setLoading] = useState<boolean | undefined>(undefined);
+  const [userId, setUserId] = useState("");
+
+  // use api calls
+  useEffect(() => {
+    if (typeof window !== "undefined" && window.localStorage) {
+      const user_Id = localStorage.getItem("userId") || "";
```
Comment (itsvick) [inline src/pages/cohorts.tsx:79]: @upendraTekdi dont use underscore in variable name

File: src/styles/style.css [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+
+.ka-thead-cell {
+  padding: 15px 20px;
+  color: #ffffff;
+  background: #8d8c8c;
+}
+
+.ka-thead-background {
+  z-index: 1;
+  background: #a8a6a6;
```
Comment (itsvick) [inline src/styles/style.css:13]: @upendraTekdi use colors from the theme variables

File: src/pages/cohorts.tsx (hunk 1 above)
Comment (upendraTekdi) [reply src/pages/cohorts.tsx:79]: done

File: src/styles/style.css (hunk 2 above)
Comment (upendraTekdi) [reply src/styles/style.css]: removed this unwanted style file  and add some css in global css file where we can not get theme colors.

**Pull request merged**